config/software/metasploit-framework.rb

```diff
@@ -40,6 +40,7 @@ def has_windows_metasploit_framework_repo?
 
 # This depends on libfuse
 whitelist_file "#{install_dir}/embedded/framework/data/exploits/CVE-2016-4557/hello"
+whitelist_file "#{install_dir}/embedded/framework/data/exploits/CVE-2023-0386/cve_2023_0386.x64.elf"
```

Comment:
This should be covered by the above line:

```
whitelist_file "#{install_dir}//embedded/framework/data/exploits/.*"
```

Which should be a regex match:

https://github.com/rapid7/omnibus/blob/c6dc312d985e85fb35b8e9f8e49b607abd91665c/lib/omnibus/software.rb#L581

Which is used with `match` in the health checks:

https://github.com/rapid7/omnibus/blame/r7_9.0.23_custom/lib/omnibus/health_check.rb#L586-L592

But for some reason it's not covered correctly; will land to unblock the deploys working - and if there's time in the future can debug further